packages/canary-web-components/src/components/ic-date-picker/ic-date-picker.tsx

```diff
@@ -262,6 +266,18 @@ export class DatePicker {
    */
   @Prop() required?: boolean = false;
 
+  /**
+   * @deprecated This prop should not be used anymore. If `true`, shows the date input control allowing entry of a date using the keyboard. If this is not provided then the component shows a calendar instead..
```

Comment:
```suggestion
   * @deprecated This prop should not be used anymore. If `true`, shows the date input control allowing entry of a date using the keyboard. If `false` then the component shows a calendar instead.
```

Just a slight change as it implied that not setting a value would cause it to display the calendar

Alternatively could change  the whole sentence to: If `false` hides the date input control and just displays the calendar

Reply:
Fixed - I went with the option of replacing the whole sentence.